### ISSUE TITLE
Add alt text to author avatars

### ISSUE THREAD

**Issue by laurelfulford** [Automattic/newspack-blocks#710]
To improve accessibility, the author avatars should have alt text. 

There is an option to add it as an attribute to both the [`get_avatar()`](https://developer.wordpress.org/reference/functions/get_avatar/) function, and the [`coauthors_get_avatar()`](https://github.com/Automattic/Co-Authors-Plus/blob/master/template-tags.php#L698) function that the block uses.
